Loosen version of Node 16 for Node.js engine in package.json

**Describe the bug** A clear and concise description of what the bug is.

Loosens the required version of Node 16 so it works on CodeSandbox. I was getting this error because they currenlty install `16.17.0`, so I opted to just make the engine check `>= 16`

```
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: connect-four@1.0.0
npm ERR! notsup Not compatible with your version of node/npm: connect-four@1.0.0
npm ERR! notsup Required: {"node":">=16.17.1"}
npm ERR! notsup Actual:   {"npm":"8.15.0","node":"v16.17.0"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /project/.cache/npm/_logs/2022-10-07T21_38_37_672Z-debug-0.log
```

![CleanShot 2022-10-07 at 17 45 03](https://user-images.githubusercontent.com/833231/194664835-3c3f58ff-f74f-42fc-8d48-584284153dce.png)

I'd like people to be able to use CodeSandbox as well, and this should unblock them.

**To Reproduce** Steps to reproduce the behavior:

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior** A clear and concise description of what you expected to
happen.

**Screenshots** If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: [e.g. iOS]
- Browser [e.g. chrome, safari]
- Version [e.g. 22]

**Smartphone (please complete the following information):**

- Device: [e.g. iPhone6]
- OS: [e.g. iOS8.1]
- Browser [e.g. stock browser, safari]
- Version [e.g. 22]

**Additional context** Add any other context about the problem here.

Maybe we could set a `.nvmrc` or something, but if it works out of the box on Codesandbox with the Node.js engine set to `>= 16`, then that's less config to manage.

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
 	},
 	"homepage": "https://github.com/nickytonline/connect-four#readme",
 	"engines": {
-		"node": ">=16.17.1"
+		"node": ">=16"
 	},
 	"devDependencies": {
 		"@playwright/test": "^1.25.0",

